improver/utilities/load.py

```diff
@@ -116,76 +106,57 @@ def load_cube(filepath, constraints=None, no_lazy_load=False,
+        enforce_coordinate_ordering(
+            cube, [y_name, x_name], anchor_start=False)
+        if no_lazy_load:
+            # Force cube's data into memory by touching the .data attribute.
+            # pylint: disable=pointless-statement
+            cube.data
 
-    # Remove metadata prefix cube attributes
-    if 'bald__isPrefixedBy' in cube.attributes.keys():
-        cube.attributes.pop('bald__isPrefixedBy')
-
-    # Ensure the probabilistic coordinates are the first coordinates within a
-    # cube and are in the specified order.
-    enforce_coordinate_ordering(
-        cube, ["realization", "percentile", "threshold"])
-    # Ensure the y and x dimensions are the last dimensions within the cube.
-    y_name = cube.coord(axis="y").name()
-    x_name = cube.coord(axis="x").name()
-    enforce_coordinate_ordering(
-        cube, [y_name, x_name], anchor_start=False)
-    if no_lazy_load:
-        # Force the cube's data into memory by touching the .data attribute.
-        # pylint: disable=pointless-statement
-        cube.data
-    return cube
+    return cubes
 
 
-def load_cubelist(filepath, constraints=None, no_lazy_load=False):
-    """Load one cube from each of the filepath(s) provided using Iris into
-    a cubelist.
+def load_cube(filepath, constraints=None, no_lazy_load=False):
+    """Load the filepath provided using Iris into a cube.  Strips off all
+    var names except for "threshold"-type coordinates, where this is different
+    from the standard or long name.
 
     Args:
         filepath (str or list):
-            Filepath(s) that will be loaded, each containing a single cube.  If
-            wildcarded, this will be expanded into a list of file names so that
-            only one cube is loaded per file.
+            Filepath that will be loaded or list of filepaths that can be
+            merged into a single cube.
         constraints (iris.Constraint, str or None):
             Constraint to be applied when loading from the input filepath.
             This can be in the form of an iris.Constraint or could be a string
             that is intended to match the name of the cube.
             The default is None.
-        no_lazy_load (bool)
+        no_lazy_load (bool):
             If True, bypass cube deferred (lazy) loading and load the whole
             cube into memory. This can increase performance at the cost of
             memory. If False (default) then lazy load.
 
     Returns:
-        iris.cube.CubeList:
-            CubeList that has been created from the input filepath given the
+        iris.cube.Cube:
+            Cube that has been loaded from the input filepath given the
             constraints provided.
     """
-    if isinstance(filepath, list) and len(filepath) == 1:
-        filepath = filepath[0]
-
-    # If the filepath is a string, then use glob, in case the str contains
-    # wildcards.
-    if isinstance(filepath, str):
-        filepaths = glob.glob(filepath)
```

Comment:
Assume we're not using the wildcard functionality at all and that's why you've removed it?  (I thought we did use this in some of the acceptance tests, but if it's not used in the suite then we don't need it.)

Reply:
`iris.load` already handles globs, so wildcards should still work. I think this was done this way due to combination of earlier rule of one cube per file and `load_cubelist` being implemented on top of `load_cube` rather than the other way around. Things simplify after we drop one cube per file rule, because we can get better alignment with how iris works.